content/1.7/back-office/login.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Login into Back-Office
+weight: 1
+---
+# LOGIN
+
+![Login UI](static/login.png)
+
+This page allows any authorized user to connect to the PrestaShop back-office. 
+
+## Top UI Section
+
+- The top section represents the Prestashop main title name and the Prestashop version, that the administrator is accessing.
+- Prestashop version - 1.7.8.0.
+
+## Middle UI Section
+
+- **Logo** - PrestaShop's logo.
+- **Shop name** - the shop name, that was defined during the installation process of Prestashop application.
+- **Email address** - email input - Email from a SuperAdmin or Employees acount must be used to log-in. 
+- **Password input** -  Password of the email account above from a SuperAdmin or Employees must be used to log-in. 
+- **Login button** - Allows access to the back office after verifying that the user is authorized to log-in.
+- **Stay logged in checkbox** - The browser will remember the logging-in status, and the administrator will not be required to input email and password in the next attempt of logging-in. The administrator will be authenticated and redirected to the dashboard instantly .
+- **I forgot my password** - specific link, in case administrator forgets the Prestashop authentication password. Once clicked, the UI instantly flips to the new UI which contains the email input. 
```

Comment:
Could you please describe all details present in this page accordingly
![image](https://user-images.githubusercontent.com/89604350/152558695-4566d930-37ef-4b4b-b9c7-39161ae10bd2.png)
Whene reading the spec I detected this issue https://github.com/PrestaShop/PrestaShop/issues/27591

Reply:
@marwachelly I have updated the specs - segmented the notifications and the main UI elements :) take a look.